msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs

```diff
@@ -24,9 +24,10 @@
 // THE SOFTWARE.
 //
 //------------------------------------------------------------------------------
+extern alias msal;
 
 using Test.Microsoft.Identity.LabInfrastructure;
-using Microsoft.Identity.Client;
+using msal::Microsoft.Identity.Client;
```

Comment:
can you please explain why this msal:: alias is now needed?

Reply:
because of namespace collisions between adal and msal, I have added integ tests for unified cache which create msal and adal client in the same test